python/lsst/daf/butler/butler.py

```diff
@@ -147,22 +148,22 @@ def makeRepo(root, config=None, standalone=False, createRegistry=True):
         config = Config(config)
         full = ButlerConfig(config)  # this applies defaults
         datastoreClass = doImport(full["datastore", "cls"])
-        datastoreClass.setConfigRoot(root, config, full)
+        datastoreClass.setConfigRoot(f"{ROOT_TAG}/datastore", config, full)
         registryClass = doImport(full["registry", "cls"])
-        registryClass.setConfigRoot(root, config, full)
+        registryClass.setConfigRoot(ROOT_TAG, config, full)
```

Comment:
I may have just not gotten far enough into the changes yet, but why the asymmetry between datastore and registry here?

Reply:
datastores write many files and where they end up and what they look like is entirely at the whims of the templates. Registry has a single file to writing it to a registry directory didn't seem worthwhile. Defaulting to a `datastore` subdirectory seems cleaner to me as a default (albeit causing pain in ci_hsc ingest).

Reply:
The asymmetry has been removed.